guides/get_started/prerequisites.md

```diff
@@ -6,6 +6,10 @@ Backpex integrates seamlessly with your existing Phoenix LiveView application, b
 
 Backpex is built on top of Phoenix LiveView, so you need to have Phoenix LiveView installed in your application. If you generate a new Phoenix application using the latest version of the `mix phx.new` generator, Phoenix LiveView is included by default.
 
+## Alpine.js
+
+Backpex uses [Alpine.js](https://alpinejs.dev/) for some interactivity. Make sure you have Alpine.js installed in your application.
```

Comment:
We should link a guide here or document the steps how to do that (`onBeforeElUpdated` and so on...).

Reply:
Do you know a good and up-to-date guide? If not, I will add the steps to our docs.

Reply:
Let's add the steps here.

Reply:
Done